babel.config.js

```diff
@@ -12,7 +12,10 @@ module.exports = {
         '@babel/preset-react',
         'react-app',
       ],
-      plugins: ['@babel/plugin-proposal-class-properties'],
+      plugins: [
+        '@babel/plugin-proposal-class-properties',
+        ['babel-plugin-styled-components', { ssr: true, displayName: false, namespace: 'sc-' }],
```

Comment:
This plugin is required to run `jest-styled-components` v7 with `styled-components` v5